azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTest.java

```diff
@@ -395,6 +381,45 @@ private void testAttempts(ExecutableFlow flow, String name, int attempt) {
     }
   }
 
+  private void waitFlowFinished(FlowRunner runner) {
+    for (int i = 0; i < 500; i++) {
+      if (runner.getExecutableFlow().isFlowFinished()) {
+        return;
+      }
+      try {
+        Thread.sleep(10);
+      } catch (InterruptedException e) {
+        // ignored
+      }
+    }
+    Assert.fail("Flow didn't finish in 5 seconds");
+  }
+
+  private void waitJobsStarted(FlowRunner runner, String[] jobs) {
+    for (int i = 0; i < 500; i++) {
+      if (checkJobsStarted(runner, jobs)) {
+        return;
+      }
+      try {
+        Thread.sleep(10);
+      } catch (InterruptedException e) {
+        // ignored
+      }
+    }
+    Assert.fail("Jobs didn't start in 5 seconds");
+  }
+
+  private boolean checkJobsStarted(FlowRunner runner, String[] jobs) {
```

Comment:
The method checks the given list of job names on the runner to see if their status is something that means that they have been at least started, if not even finished.

Do you mean that javadoc should be added?

Personally I feel that this method name should be clear enough, and having a glance at the code is probably better than my sorry attempt to explain it above :)